code/labflow-backend/src/main/java/com/thomasmore/blc/labflow/service/PdfGeneratorService.java

```diff
@@ -71,6 +69,103 @@ public byte[] generateLabelPdf(Staal staal) throws DocumentException {
         return out.toByteArray();
     }
 
+    public byte[] generateLabelPdfv2(Staal staal) throws DocumentException {
```

Comment:
GenerateLabelPdf heeft geen usages maar staat nog altijd in de PdfGeneratorService.
Ik zou v2 weghalen uit de naam en de eerste versie verwijderen.

Reply:
oke doe ik!